handlers/pricing.go

```diff
@@ -113,7 +113,7 @@ func MobilePricing(w http.ResponseWriter, r *http.Request) {
 func PricingPost(w http.ResponseWriter, r *http.Request) {
 	err := r.ParseForm()
 	if err != nil {
-		logger.Errorf("error parsing form: %v", err)
+		utils.LogError(err, "error parsing form", 0)
 		utils.SetFlash(w, r, "pricing_flash", "Error: invalid form submitted")
 		logger.Errorf("error parsing pricing request form for %v route: %v", r.URL.String(), err)
```

Comment:
What about this line? Why are there two error logs anyways?

Reply:
I don't know, just changed to utils for better location of the error.
Already asked in Discord to remove the "error parsing form" completely, as I see no point in the info.